src/components/Filters/components/Input/Input.tsx

```diff
@@ -32,9 +35,9 @@ export const StyledInput = styled.input`
     border-color: ${getFormStyle('focusBorderColor')};
   }
 
-  &:invalid {
-    ${stateStyles}
-    border-color: ${getFormStyle('invalidBorderColor')};
+  &:disabled {
```

Comment:
Do we need this styles for disabled state? I don't think we are disabling the input